frontend/src/features/idir/search/table/CompanySearchResultColumnDef.tsx

```diff
@@ -26,8 +26,8 @@ export const CompanySearchResultColumnDef: MRT_ColumnDef<CompanyProfile>[] = [
     sortingFn: "alphanumeric",
   },
   {
-    accessorKey: "migratedClientHash",
-    header: "Client Number",
+    accessorKey: "clientNumber",
```

Comment:
Aren't we negotiating the clientNumber as a hash?

Reply:
in the `read-company` DTO from the service we're supplying both:
```
  @AutoMap()
  @ApiProperty({
    description: 'The ORBC client number.',
    example: '1234',
  })
  clientNumber: string;

  @AutoMap()
  @ApiProperty({
    description: 'The migrated client hash.',
    example: '8db3c29a00c728c0a3c31b91662a4ba4280e50ac6bfc388a7765d1bcb46845da',
    required: false,
  })
  migratedClientHash?: string;
  ```

Reply:
ok, thanks!